targets/ipq806x

```diff
@@ -1,7 +1,31 @@
-ATH10K_PACKAGES='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca99x0 ath10k-firmware-qca99x0-ct'
+# The QCA9980 is discontinued from Qualcomm. It didn't receive any firmware-update since >4 years.
+# See https://github.com/kvalo/ath10k-firmware/tree/master/QCA99X0/hw2.0
+# 802.11s was never implemented for the chips firmware. It is most likely broken forever.
```

Comment:
chip's
i'm also not sure if "It will most likely be broken forever." is better?

Reply:
fixed